<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VarsWidget</class>
 <widget class="QWidget" name="VarsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vars</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="table"/>
   </item>
   <item>
    <widget class="QWidget" name="toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0,10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="add">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="del">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="reload">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_Watch">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/edit-add.svg</normaloff>:/app_data/icons/edit-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Watch</string>
   </property>
   <property name="toolTip">
    <string>Add watched expression</string>
   </property>
  </action>
  <action name="actionRemove_Watch">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/remove.svg</normaloff>:/app_data/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Watch</string>
   </property>
   <property name="toolTip">
    <string>Remove watched expression</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/reload.svg</normaloff>:/app_data/icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
